_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP? Why is it so important to do a pre-GSoC analysis
+and design a proposal as early as possible?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the **coala** community
+to discuss the new *major features* that the contributor intends to implement
+during the GSoC duration. The proposal is designed before the implementation of
```

Comment:
```during GSoC``` would convey the same message as ```during the GSoC duration```

Reply:
Earlier it was `during GSoC` but @utkarsh2102 suggested a change 
He mentioned that other docs contains either `GSoC duration` or  `GSoC time`
So to main consistency I changed
Should I go for a change or not ?

Reply:
I think ```during GSoC``` is completely fine. @shashank-b what do you have to say about this?

Reply:
Yeah, I agree, both seem fine. `during GSoC` seems more commonly used.